org.eclipse.lsp4xml.extensions.maven/src/main/java/org/eclipse/lsp4xml/extensions/maven/MavenRepositoryCache.java

```diff
@@ -17,71 +17,6 @@
 import org.eclipse.aether.repository.RemoteRepository;
 
 public class MavenRepositoryCache {
```

Comment:
Should we just delete this class as it's logic has moved to `RemoteRepositoryIndexSearcher`?

Reply:
Sure.